src/styles/stars.css

```diff
@@ -0,0 +1,36 @@
+.stars {
+  display: flex;
+  align-items: end;
+}
+
+.stars__star {
+  background-image: url("../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url("../images/star-active.svg");
+}
+
+.stars::after {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  content: "Reviews: 5";
+  flex-grow: 1;
+  text-align: end;
+}
```

Comment:
it should be `html` content, and it should not belong to stars block

Reply:
![image](https://user-images.githubusercontent.com/90213755/226931291-964f60e0-f9fd-4a35-967b-9a58d4dd13e0.png)
![image](https://user-images.githubusercontent.com/90213755/226931349-01d6ebd4-986a-4247-b3dc-9ae438a012b0.png)
i have this error when i try to make block